_data-prepper/pipelines/configuration/processors/processors.md

```diff
@@ -10,5 +10,7 @@ nav_order: 25
 
 Processors perform some action on your data: filter, transform, enrich, etc.
 
-Prior to Data Prepper 1.3, Processors were named Preppers. Starting in Data Prepper 1.3, the term Prepper is deprecated in favor of Processor. Data Prepper will continue to support the term "Prepper" until 2.0, where it will be removed.
-{: .note }
\ No newline at end of file
+Prior to Data Prepper 1.3, processors were named preppers. Starting in Data Prepper 1.3, the term Prepper is deprecated in favor of Processor. Data Prepper will continue to support the term "Prepper" until 2.0, where it will be removed.
+{: .note }
+
+<!---Tech/Editorial: Do we need to keep this page now that Data Prepper is past 2.0?--->
```

Comment:
@graytaylor0 Is this information still relevant now that Data Prepper has reached version 2.0?